apps/client/src/stores/models.media.edit.spec.ts

```
@@ -0,0 +1,171 @@
+  };
+
+  beforeEach(() => {
+    // mock first media file
+    mocks.getMediaInfo.mockResolvedValueOnce({ data: mediaInfo1 });
+    mocks.download.mockResolvedValueOnce({ status: 200, data: blob1 });
+    mocks.mockObjectURL.mockReturnValueOnce("mockObjectURL1");
+    // mock second media file
+    mocks.getMediaInfo.mockResolvedValueOnce({ data: mediaInfo2 });
+    mocks.download.mockResolvedValueOnce({ status: 200, data: blob2 });
+    mocks.mockObjectURL.mockReturnValueOnce("mockObjectURL2");
+  });
```

Biome (2.1.2)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)